src/engine/sidechain/shoutconnection.cpp

```diff
@@ -881,53 +887,65 @@ void ShoutConnection::updateMetaData() {
         }
     } else {
         // Otherwise we might use static metadata
-        // If we use static metadata, we only need to call the following line once
-        if (m_custom_metadata && !m_firstCall) {
-
+        // If we use static metadata, we only need to call this only once
+        if (m_custom_metadata && m_firstCall) {
             // see comment above...
             if (!m_format_is_mp3 && m_protocol_is_icecast2) {
-            	setFunctionCode(12);
+                kLogger.info() << "Updating static metadata for"
+                               << profile()->getProfileName()
+                               << ": artist:" << m_customArtist
+                               << "| title:" << m_customTitle;
+                setFunctionCode(12);
                 insertMetaData("artist", encodeString(m_customArtist).constData());
                 insertMetaData("title", encodeString(m_customTitle).constData());
             } else {
-                QByteArray baCustomSong = encodeString(m_customArtist.isEmpty() ? m_customTitle : m_customArtist + " - " + m_customTitle);
+                const QString songStr = m_customArtist.isEmpty()
+                        ? m_customTitle
+                        : m_customArtist + " - " + m_customTitle;
+                kLogger.info()
+                        << "Updating static metadata for"
+                        << profile()->getProfileName()
+                        << ": song:" << songStr;
+                const QByteArray baCustomSong = encodeString(songStr);
                 insertMetaData("song", baCustomSong.constData());
             }
 
             setFunctionCode(13);
             shout_set_metadata(m_pShout, m_pShoutMetaData);
-            m_firstCall = true;
+            m_firstCall = false;
```

Comment:
So if I understand correctly, this variable used to store `has done first call` and now will store `is first call`. How about making that clear and naming `isFirstCall`?